commands/serverless-extra.go

```diff
@@ -36,7 +36,7 @@ to choose your sample language for `+"`"+`doctl serverless init`+"`"+`.`,
 	AddStringFlag(create, "language", "l", "javascript", "Language for the initial sample code")
 	AddBoolFlag(create, "overwrite", "", false, "Clears and reuses an existing directory")
 
-	deploy := CmdBuilder(cmd, RunServerlessExtraDeploy, "deploy <directories>", "Deploy a functions project to your functions namespace",
+	deploy := CmdBuilder(cmd, RunServerlessExtraDeploy, "deploy <directory>", "Deploy a functions project to your functions namespace",
```

Comment:
An unrelated fix.   Only one project (directory) can be deployed at a time so the original text was misleading.